drivers: display: extend support for Renesas RA GLCDC (#84080)
zephyrproject-rtos/zephyr: boards/shields/rtk7eka6m3b00001bu/doc/index.rst

.. _rtk7eka6m3b00001bu:

RTK7EKA6M3B00001BU Display

> ###############################

Author: I fixed it

Overview
********

> The Graphics Expansion Board includes a 4.3-inch 480 x 272 pixel TFT color LCD with a

Reviewer: ```suggestion
The Graphics Expansion Board includes a 4.3-inch 480x272 pixel TFT color LCD with a
```

Author: I fixed it

capacitive touch overlay.

> This display uses a 40 pin connector header.

Reviewer: ```suggestion
This display uses a 40-pin connector header.
```

Author: I fixed it

Pins Assignment of the Renesas RTK7EKA6M3B00001BU Display
=========================================================

+-----------------+--------------------------+
| Connector Pin   | Function                 |
+=================+==========================+
| 1               | Display backlight enable |
+-----------------+--------------------------+
| 2               | Touch ctrl I2C SDA       |
+-----------------+--------------------------+
| 3               | External interrupt       |
+-----------------+--------------------------+
| 4               | Touch ctrl I2C SCL       |
+-----------------+--------------------------+
| 6               | Display reset            |
+-----------------+--------------------------+

Hardware Requirements:
**********************

> Supported Renesas RA boards: EK-RA8D1
> - 1 x RA Board

Reviewer: ```suggestion
Supported Renesas RA boards: EK-RA8D1

- 1 x RA Board
```

Author: I fixed it

- 1 x Micro USB cable

Hardware Configuration:
***********************

The Graphics Expansion Port (J57) connects the EK-RA8D1 board to the Graphics Expansion Board
supplied as part of the kit.

Set the configuration switches (SW1) as below to avoid potential failures.

> 		+-------------+-------------+--------------+------------+------------+------------+-------------+-----------+
> 		| SW1-1 PMOD1 | SW1-2 TRACE | SW1-3 CAMERA | SW1-4 ETHA | SW1-5 ETHB | SW1-6 GLCD | SW1-7 SDRAM | SW1-8 I3C |
> 		+-------------+-------------+--------------+------------+------------+------------+-------------+-----------+
> 		|     OFF     |     OFF     |      OFF     |     OFF    |     OFF    |     ON     |     ON      |    OFF    |
> 		+-------------+-------------+--------------+------------+------------+------------+-------------+-----------+

Reviewer: no tabs, and please swap the table so that it's not unnecessarily wide. FWIW a simple bullet list probably works just fine

```suggestion
+----------------+-----+
| SW1-1 PMOD1    | OFF |
+----------------+-----+
| SW1-2 TRACE    | OFF |
+----------------+-----+
| SW1-3 CAMERA   | OFF |
+----------------+-----+
| SW1-4 ETHA     | OFF |
+----------------+-----+
| SW1-5 ETHB     | OFF |
+----------------+-----+
| SW1-6 GLCD     | ON  |
+----------------+-----+
| SW1-7 SDRAM    | ON  |
+----------------+-----+
| SW1-8 I3C      | OFF |
+----------------+-----+
```

Author: I removed the tabs but kept the horizontal table because it is consistent with the official board's documentation on the Renesas website and is better for the user experience because it corresponds 1:1 to the switch positions on the board.

Reviewer: I see what you mean, fair enough :)
